src/main/scala/com/gu/zuora/creditor/ZuoraAPIClients.scala

```diff
@@ -22,25 +16,31 @@ trait ZuoraRestClient {
   def makeRestPOST(path: String)(commandJSON: SerialisedJson): SerialisedJson
 }
 
+final case class ZuoraRestConfig(
+                                  zuoraApiHost: String,
+                                  zuoraApiAccessKeyId: String,
+                                  zuoraApiSecretAccessKey: String
+                                )
 
-object ZuoraAPIClientsFromEnvironment extends LazyLogging {
 
-  private def decryptSecret(cyphertext: String) = {
-    if (getenv("SkipSecretDecryption") == "true") {
-      cyphertext
-    } else {
-      val encryptedKey = ByteBuffer.wrap(Base64.decode(cyphertext))
-      val client = AWSKMSClientBuilder.defaultClient
-      val request = new DecryptRequest().withCiphertextBlob(encryptedKey)
-      val plainTextKey = client.decrypt(request).getPlaintext
-      new String(plainTextKey.array(), Charset.forName("UTF-8"))
+object ZuoraAPIClientFromParameterStore extends LazyLogging {
+
+  private val loadConfig = {
+    val identity = AppIdentity.whoAmI(defaultAppName = "zuora-creditor")
```

Comment:
The lambdas in support-service-lambdas load their config from a shared location.

I realise this isnt in support-service-lambdas but did you consider using that shared config, are these that same creds as support-service-lambdas?

If they are shared its less work to roll them.

Reply:
no the creds are different it is for `subscriptions` not `membership`, the are not oauth creds

Reply:
You don't need oauth with Zuora, it just puts the url, email and password as HTTP headers named as below:
```
      zuoraApiHost = config.getString("zuoraApiHost"),
      zuoraApiAccessKeyId = config.getString("zuoraApiAccessKeyId"),
      zuoraApiSecretAccessKey = config.getString("zuoraApiSecretAccessKey")
```

Reply:
So you should be able to share.

Reply:
if we're going to share creds, would it be worth actually moving this project into `support-service-lambdas`??

Reply:
(FWIW I don't really like the idea of this project reading config from S3 which has `support-service-lambdas` in its path if it's not actually part of that monorepo)

Reply:
i don't like the idea of reading from S3 either
AWS SSM should be the way to go, as this is what it is for :) 

sharing Creds is another concept that can cause bugs in the future